src/main/java/com/googlecode/loosejar/output/CSVFormatSummarizer.java

```diff
@@ -0,0 +1,52 @@
+package com.googlecode.loosejar.output;
+
+import java.util.List;
+import java.util.Map;
+
+import com.googlecode.loosejar.ClassLoaderAnalyzer;
+import com.googlecode.loosejar.JarArchive;
+
+public class CSVFormatSummarizer implements Summarizer {
```

Comment:
Hmm... That's not classical CSV format. I mean, in a CSV format all fields must be quoted, quotes escaped, etc. There are libraries that do that. It is probably ok to let stand like that, but I would have used a library to generate the CSV. Have you tried importing the result into Excel? Does it even work without the quotes around the fields?